.github/workflows/test_frameworks.yml

```diff
@@ -112,21 +112,18 @@ jobs:
       - uses: actions/checkout@v3
         with:
           repository: tiangolo/fastapi
-          ref: 0.63.0
+          ref: 0.75.0
           path: fastapi
       #This step installs Flit, a way to put Python packages and modules on PyPI (More info at https://flit.readthedocs.io/en/latest/)
       - name: Install Flit
         run: pip install flit
       #Installs all dependencies needed for FastAPI
       - name: Install Dependencies
         run: flit install --symlink
-      #Reverts SQLalchemy to v1.3.24. Newest version breaks tests due to RowProxy module being deprecated in latest v1.4.11 More info: (https://docs.sqlalchemy.org/en/14/changelog/migration_14.html#rowproxy-is-no-longer-a-proxy-is-now-called-row-and-behaves-like-an-enhanced-named-tuple)
-      - name: Downgrade SQLalchemy
-        run: pip install --force-reinstall SQLAlchemy==1.3.24
       - name: Inject ddtrace
         run: pip install ../ddtrace
       - name: Test
-        run: ddtrace-run pytest tests
+        run: ddtrace-run pytest -p no:warnings tests
 
   flask-testsuite-1_1_4:
```

Comment:
is this still an issue with newer versions of flask? should we just bump the version?

Reply:
MarkupSafe 2.1.0 was released on Feb 18, while the last Flask release dates to Feb 14, so I think there might be chances we'd still need this fix at this point in time.